.github/workflows/public_docker_images.yml

```diff
@@ -9,7 +9,7 @@ on:
   pull_request:
     branches: main
   schedule:
-    - cron: '0 2 29 * *'  # At 02:00 on day-of-month 29
+    - cron: '0 2 1 */3 *'  # At 02:00 on the 1st day of every 3rd month
```

Comment:
@odulcy-mindee adjusted a bit we have already 200+ images in the registry I think every month + releases was to much :sweat_smile:

Reply:
Haha, ok, that's fine

Reply:
I'm glad to see it works again

Reply:
Yeah :D